From: Ben Hutchings <ben@decadent.org.uk>
Date: Thu, 26 Oct 2017 11:59:43 +0200
Subject: SCSI: Avoid ABI change in 4.13.6
Forwarded: not-needed

Hide the new bitfield from genksyms, as it's using what used to be a
padding bit.

---
--- a/include/scsi/scsi_device.h
+++ b/include/scsi/scsi_device.h
@@ -182,7 +182,10 @@ struct scsi_device {
 	unsigned no_dif:1;	/* T10 PI (DIF) should be disabled */
 	unsigned broken_fua:1;		/* Don't set FUA bit */
 	unsigned lun_in_cdb:1;		/* Store LUN bits in CDB[1] */
+#ifndef __GENKSYMS__
 	unsigned unmap_limit_for_ws:1;	/* Use the UNMAP limit for WRITE SAME */
+	/* 19 unused bits */
+#endif
 
 	atomic_t disk_events_disable_depth; /* disable depth for disk events */
 
